spring-boot/admin/src/main/java/eu/coatrack/admin/service/GitService.java

```diff
@@ -104,6 +107,7 @@ public void addProxy(Proxy proxy) throws GitAPIException, URISyntaxException, Fi
         PrintWriter writer = new PrintWriter(resultDir + "/ygg-proxy-" + proxy.getId() + ".yml", "UTF-8");
         writer.println("proxy-id: " + proxy.getId());
         writer.println("ygg.admin.api-base-url: " + adminApiBaseUrlForGateway);
+        writer.println("ygg.admin.server.url: " + adminServerUrl);
```

Comment:
We already have the very similar parameter `ygg.admin.api-base-url`. I was wondering if we could avoid to add a new config parameter? Seems redundant.

Also this parameter is not included in existing gateway configs, so when re-downloading an existing gateway, it will not start up.

Reply:
@ChristophBaierATB I checked and there is more 2 classes which also includes the ygg.admin.server.url, please check my review at the end

Reply:
Done. It is removed.